README.md

```diff
@@ -1,97 +1,3 @@
 # FVTool: Finite volume toolbox for Matlab
+This a fork of simulkade/FVTool for personal tests.
```

Comment:
Hi @LioSergent , could you keep the old readme file and add a paragraph about the new features?